install_config/install/advanced_install.adoc

```diff
@@ -931,8 +931,8 @@ link:#verifying-the-installation[Verifying the Installation].
 == Verifying the Installation
 
 After the installer completes, you can verify that the master is started and
-nodes are registered and reporting in *Ready* status by running the following as
-*root*:
+nodes are registered and reporting in *Ready* status by sshing into the master
+host and running the following as *root*:
```

Comment:
How about:

"After the installer completes, verify that the master is started and nodes are registered and reporting in Ready status. On the master host, run the following as root:"